src/pages/experiments/[id].tsx

```diff
@@ -62,53 +95,66 @@ export default function Experiment() {
   const canModify = experiment.data?.access.canModify ?? false;
 
   return (
-    <AppShell title={experiment.data?.label}>
-      <VStack h="full">
-        <Flex
-          px={4}
-          py={2}
-          w="full"
-          direction={{ base: "column", sm: "row" }}
-          alignItems={{ base: "flex-start", sm: "center" }}
-        >
-          <Breadcrumb flex={1}>
-            <BreadcrumbItem>
-              <Link href="/experiments">
-                <Flex alignItems="center" _hover={{ textDecoration: "underline" }}>
-                  <Icon as={RiFlaskLine} boxSize={4} mr={2} /> Experiments
-                </Flex>
-              </Link>
-            </BreadcrumbItem>
-            <BreadcrumbItem isCurrentPage>
-              {canModify ? (
-                <Input
-                  size="sm"
-                  value={label}
-                  onChange={(e) => setLabel(e.target.value)}
-                  onBlur={onSaveLabel}
-                  borderWidth={1}
-                  borderColor="transparent"
-                  fontSize={16}
-                  px={0}
-                  minW={{ base: 100, lg: 300 }}
-                  flex={1}
-                  _hover={{ borderColor: "gray.300" }}
-                  _focus={{ borderColor: "blue.500", outline: "none" }}
-                />
-              ) : (
-                <Text fontSize={16} px={0} minW={{ base: 100, lg: 300 }} flex={1}>
-                  {experiment.data?.label}
-                </Text>
-              )}
-            </BreadcrumbItem>
-          </Breadcrumb>
-          <HeaderButtons />
-        </Flex>
-        <ExperimentSettingsDrawer />
-        <Box w="100%" overflowX="auto" flex={1}>
-          <OutputsTable experimentId={router.query.id as string | undefined} />
-        </Box>
-      </VStack>
-    </AppShell>
+    <>
+      {stats && (
+        <Head>
+          <meta property="og:title" content={stats.experimentLabel} />
+          <meta
+            property="og:image"
+            content={`/api/experiments/og-image?experimentLabel=${stats.experimentLabel}&variantsCount=${stats.promptVariantCount}&scenariosCount=${stats.testScenarioCount}`}
+          />
+          <meta property="og:image:height" content="630" />
```

Comment:
You can use the `key` property on all of these to make it so we don't have duplicate tags in the rendered HTML, which might confuse some processors. https://nextjs.org/docs/pages/api-reference/components/head

Reply:
Good idea